packages/components/src/styles/components/dropdown.scss

```
@@ -33,6 +33,7 @@ $hds-dropdown-toggle-border-radius: $hds-button-border-radius;
   align-items: center;
   justify-content: center;
   padding: 1px;
+  color: var(--token-color-foreground-primary); // does not have secondary & primary variants so we use the primary color
```

I would keep the comment even more generic
```suggestion
  // this variant mimics the secondary button, but not entirely (eg. it doesn't have an elevation shadow)
  // so we need to provide the same color values but without using the `hds-button-color-secondary()` mixin
  color: var(--token-color-foreground-primary);
```
(feel free to edit/reword)